gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -31,12 +31,15 @@
  * <refsect2>
  * <title>Example launch line with simple if condition</title>
  * gst-launch ... (some tensor stream) !
- *      tensor_if compared_value=A_VALUE compared_value_option=3:4:2:5,0
- *      operator=RANGE_INCLUSIVE
- *      supplied_values=10,100
- *      then=PASSTHROUGH
- *      else=FILL_WITH_FILE else_option=${path_to_file}
- *    ! (tensor stream) ...
+ *      tensor_if name=tif \
+ *                compared-value=A_VALUE compared-value_option=3:4:2:5,0 \
+ *                operator=RANGE_INCLUSIVE \
+ *                supplied-values=10,100 \
+ *                then=PASSTHROUGH \
```

Comment:
PASSTHROUGH == TENSOR_PICK at src.0? (need to document it, too)

Reply:
src_0 is for TRUE action.